### PR TITLE
Feature/90 add linting test tasks

### PR DESCRIPTION
**Pull request by paulo-teran-jalafund**
[AB#90](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/90)

### CODE REVIEW

**Review by fernando-maceda-jalafund — APPROVED** [review]
LGTM!

**Comment by martin-callejas-jalafund** [comment]
LGTM, get your approvals and let us know to add the variables to Azure before merging ok?

**Review by niver-copa-jalafund — APPROVED** [review]

**Review by lina-ramirez-jalafund — APPROVED** [review]

**Comment by paulo-teran-jalafund** [comment]
Hey @martin-callejas-jalafund I got my 3 approvals, and my variables are already in Azure, check it out please, and let me know if I can proceed with the merge, thanks!

**Review by martin-callejas-jalafund — APPROVED** [review]

**Pull request merged**
